docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
+
+   ```sql
+   CREATE TABLE IF NOT EXISTS library.book_tracker (book_id text PRIMARY KEY, item_count int, genre text, status text, last_update_timestamp timestamp);
+   ```
+
+## Insert data
+
+Use `cqlsh` to write data in the `book_tracker` table, for example:
+
+```sql
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('01234', 6, 'true_crime', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('56789', 13, 'romance', 'unavailable', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('12345', 55, 'spirituality_religions', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('67890', 3, 'adventure', 'unavailable', toTimeStamp(now()));
+```
+
+:::tip
+For importing large amount of data or for batch data load, see:
+
+- [Use DSBULK to load, unload, and count data](/docs/products/cassandra/howto/use-dsbulk-with-cassandra)
+- [Migrate to Aiven with the ZDM Proxy](/docs/products/cassandra/howto/zdm-proxy)
+- [Migrate to Aiven with `sstableloader`](https://aiven.io/blog/aiven-now-supports-sstableloader)
 
 :::
+
+## Read data
+
+Use `cqlsh` to read data from the `book_tracker` table, for example:
+
+- ```sql
+  SELECT * FROM library.book_tracker;
+
+  book_id | genre                  | item_count | last_update_timestamp           | status
+  --------+------------------------+------------+---------------------------------+------------
+  01234   |             true_crime |          6 | 2024-07-31 11:21:14.263000+0000 |   available
+  12345   | spirituality_religions |         55 | 2024-07-31 11:24:20.373000+0000 |   available
+  67890   |              adventure |          3 | 2024-07-31 11:24:21.957000+0000 | unavailable
+  56789   |                romance |         13 | 2024-07-31 11:24:20.328000+0000 | unavailable
+  ```
+
+- ```sql
+  SELECT * FROM library.book_tracker where status = 'available' ALLOW FILTERING;
+
+  book_id | genre                  | item_count | last_update_timestamp           | status
+  --------+------------------------+------------+---------------------------------+----------
+  01234   |             true_crime |          6 | 2024-07-31 11:21:14.263000+0000 | available
+  12345   | spirituality_religions |         55 | 2024-07-31 11:24:20.373000+0000 | available
+  ```
+
+- ```sql
+  SELECT book_id, genre FROM library.book_tracker WHERE item_count > 10 AND status = 'unavailable' ALLOW FILTERING;
+
+  book_id | genre
+  --------+--------
+  56789   | romance
+  ```
+
+## Related pages
+
+- [Supported Aiven for Apache Cassandra versions](/docs/platform/reference/eol-for-major-versions#h_0f2929c770)
+- [Aiven for Apache Cassandra backups](/docs/platform/concepts/service_backups#aiven-for-apache-cassandra)
```

Comment:
(could be a separate PR) in that backup article, maybe mention that `backup_hour` and `backup_minute` are available for Cassandras and allow the customer to set _when_ the backup starts within the 24 hours? something a-la "if you want to avoid spending extra resources on backups during peak hours, you can set backup_hour in advanced user_config to move the backup outside the peak hours"

Reply:
thanks! excellent point! updating on this PR...